tests/php/test-class-wc-connect-api-client.php

```diff
@@ -56,49 +80,56 @@ public function test_build_shipment_contents_simple_product() {
 
 		$actual = $this->api_client->build_shipment_contents( array( 'contents' => WC()->cart->get_cart() ) );
 
-		$expected = array( array(
-			'height'     => 5,
-			'product_id' => $product_id,
-			'length'     => 7,
-			'width'      => 6,
-			'quantity'   => 1,
-			'weight'     => 2,
-		) );
+		$expected = array(
+			array(
+				'height'     => 5,
+				'product_id' => $product_id,
+				'length'     => 7,
+				'width'      => 6,
+				'quantity'   => 1,
+				'weight'     => 2,
+			),
+		);
 
 		$this->assertEquals( $actual, $expected );
 	}
 
+	/**
+	 * Test variable product
+	 */
 	public function test_build_shipment_contents_variable_product() {
 		$this->product = WC_Helper_Product::create_variation_product();
 
 		$all_variations     = $this->product->get_available_variations();
-		$first_variation_id = $all_variations[ 0 ][ 'variation_id' ];
+		$first_variation_id = $all_variations[2]['variation_id'];
```

Comment:
Using the 2nd variation so that it has "colour" and "number". For some reasons, without these 2 fields, the test would fail (https://github.com/Automattic/woocommerce-services/pull/2216#issuecomment-708066356).